R/lim.R

```diff
@@ -15,6 +15,9 @@ lim_args = function(datapoints, xlim, ylim, type) {
   if (identical(type, "boxplot")) {
     xlim = xlim + c(-0.5, 0.5)
   }
+  
+  if (!is.null(xaxb)) xlim = range(c(xlim, xaxb))
```

Comment:
Shold this go inside the `if(){}` above? If `xlim` is supplied explicitly, we probably probably don't want to override it with `xaxb`.

Reply:
Yeah, I'm on the fence with this. I guess making an explicit exception for user-supplied `x/ylim` makes the most sense, though. Will action when I'm back at my laptop.